src/register/ucause.rs

```diff
@@ -0,0 +1,60 @@
+//! ucause register
+
+pub use crate::register::mcause::{Interrupt, Exception, Trap};
```

Comment:
Same here. Is there any information about `scause` register in specs?

Reply:
Asking around now, seems like it's blatantly missing.

Reply:
Made ucause only readable as bits since the `N` extension doesn't seem to be finalized.